lib/KListWithOverflow.vue

```diff
@@ -103,26 +103,43 @@
       this.$watch('elementWidth', this.throttledSetOverflowItems);
     },
     methods: {
+      getWidth(element) {
+        if (!element) {
+          return 0;
+        }
+        return element.getBoundingClientRect().width;
```

Comment:
One thing to flag, getBoundingClientRect forces a reflow, so this can cause performance issues. More info here: https://toruskit.com/blog/how-to-get-element-bounds-without-reflow/

Reply:
Thanks Richard, I will batch all getWidth calls before changing anything in the DOM